apps/meteor/server/models/raw/Subscriptions.ts

```diff
@@ -113,9 +113,16 @@ export class SubscriptionsRaw extends BaseRaw<ISubscription> implements ISubscri
 		};
 
 		const update = {
+			...(readThreads && {
+				$unset: {
+					tunread: 1,
+					tunreadUser: 1,
+					tunreadGroup: 1,
+				} as const,
+			}),
 			$set: {
 				open: true,
-				alert,
+				alert: false,
```

Comment:
always updating `alert` to `false` may let the room in a weird state, having unread threads but not showing the room as unread, like in the image below:
![image](https://user-images.githubusercontent.com/8591547/195839931-f318f3bb-2b90-4d3d-acf9-39d18f2a5814.png)

Reply:
updated!